docs/topics/migrations.txt

```diff
@@ -478,6 +478,13 @@ You can pass a second callable to
 want executed when migrating backwards. If this callable is omitted, migrating
 backwards will raise an exception.
 
+.. note::
+
+    When migrating data across models contained in two different apps,
+    ``dependencies`` should include the latest migration in the target app,
+    otherwise you will get an error: "LookupError: No installed app with label
```

Comment:
Actually, I think since bbbed99f6260a8b3e65cb990e49721b1ce4a441b this does not necessarily happen. Only if the migration you are about to apply relies on a migration that hasn't been applied yet.

In other words: you have a data migration `a.0002` and a schema migration `b.0002`, where each only depends on migration the `0001` of its own app, you cannot use a model from `B` inside `a.0002`. But if `b.0002` is your data migration this would work (I think) simply because the migrations are sorted alphabetically by their app_label. That does not mean users should define an explicit dependency, though.